<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-26T02:07:12 EEST" name="Default suite" finished-at="2024-06-26T02:07:20 EEST" duration-ms="8411">
    <groups>
    </groups>
    <test started-at="2024-06-26T02:07:12 EEST" name="Default test" finished-at="2024-06-26T02:07:20 EEST" duration-ms="8411">
      <class name="myTestcases">
        <test-method is-config="true" signature="setUp()[pri:0, instance:myTestcases@4f936da8]" started-at="2024-06-26T02:07:12 EEST" name="setUp" finished-at="2024-06-26T02:07:12 EEST" duration-ms="501" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="addData()[pri:1, instance:myTestcases@4f936da8]" started-at="2024-06-26T02:07:12 EEST" name="addData" finished-at="2024-06-26T02:07:13 EEST" duration-ms="120" status="FAIL">
          <exception class="java.sql.SQLIntegrityConstraintViolationException">
            <message>
              <![CDATA[Duplicate entry '77' for key 'customers.PRIMARY']]>
            </message>
            <full-stacktrace>
              <![CDATA[java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '77' for key 'customers.PRIMARY'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:109)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:113)
at com.mysql.cj.jdbc.StatementImpl.executeUpdateInternal(StatementImpl.java:1454)
at com.mysql.cj.jdbc.StatementImpl.executeLargeUpdate(StatementImpl.java:2185)
at com.mysql.cj.jdbc.StatementImpl.executeUpdate(StatementImpl.java:1331)
at myTestcases.addData(myTestcases.java:67)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.sql.SQLIntegrityConstraintViolationException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addData -->
        <test-method signature="updatData()[pri:2, instance:myTestcases@4f936da8]" started-at="2024-06-26T02:07:13 EEST" name="updatData" finished-at="2024-06-26T02:07:13 EEST" duration-ms="756" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatData -->
        <test-method signature="getData()[pri:3, instance:myTestcases@4f936da8]" started-at="2024-06-26T02:07:13 EEST" name="getData" finished-at="2024-06-26T02:07:20 EEST" duration-ms="6643" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getData -->
        <test-method signature="deleteData()[pri:4, instance:myTestcases@4f936da8]" started-at="2024-06-26T02:07:20 EEST" name="deleteData" finished-at="2024-06-26T02:07:20 EEST" duration-ms="329" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteData -->
      </class> <!-- myTestcases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
